Bot.js

```diff
@@ -4,59 +4,73 @@ const client = new Discord.Client();
 
 //Constant for commands file
 const {commands} = require('./src/');
-
-var commandChannels = [];
+const {prefix, token} = require('./config/Bot.json')
+const blacklist = [];
 
 //On ready, output logged in message
-client.on('ready', () =>
-{
-  console.log(`Logged in as ${client.user.tag}`);
-
-  //Gets command channels where you put bot commands in
-  commandChannels = Commands.getCommandChannels(client);
+client.once('ready', () => {
```

Comment:
Difference between 'on' and 'once'?

Reply:
`on`  keeps listening for "ready",  while `once` only listens for it once.

Reply:
`on`  keeps listening for "ready",  while `once` only listens for it once.

Reply:
So why only listen for it once? Just because the client should only ready once? I get it, I'm just not sure why it's important,

Reply:
Got it, thanks